### DISCUSSION TITLE
Other payment than Paypal

### DISCUSSION THREAD

**Discussion by MAAREKANTONY** [Labs64/NetLicensing-Community#99] [Questions] [answered]
Hello support

I would like to subscribe for monthly licence but when I try to pay I am redirected to paypal.
The problem is that my company is hosted in UK but I am french.
Paypal is asking for a UK phone number and I don't have any.

Is there another way to pay ?

Thanks

**Comment by r-brown** [comment, marked as answer]
Hello @MAAREKANTONY ,

Currently there are following payment methods supported in NetLicensing: PayPal and Stripe.

However, any other eCommerce system can be used as an eComm frontend and connected to NetLicensing using [NetLicensing Gateway](https://netlicensing.io/wiki/gateway) module.

NetLicensing Gateway is an open-source and easily extendable module.
Please indicate which eCommerce system would suit your regulatory requirements, so we can verify on-boarding possibility here.
